chutney/ui/src/app/shared/components/layout/right-menu/chutney-right-menu.component.scss

```diff
@@ -1,7 +1,15 @@
+@import "~bootswatch/dist/flatly/variables";
+
 .nav-item {
     white-space: nowrap;
 }
 
 .nav-link:hover {
     color: var(--bs-primary)!important;
 }
+
+.chutney-nav-menu {
```

Comment:
I would use the definitions from **app_main_inner** and **container-fluid** classes for the margins and top declarations.

```
/* from app_main_inner and container-fluid classes */
margin-left: calc(var(--bs-gutter-x)* -0.5);
margin-right: calc(var(--bs-gutter-x)* -0.5);
margin-top: -1.5rem;
top: -1.5rem;
```